sources/Application/Instruments/WavFileWriter.h

```diff
@@ -46,8 +47,8 @@ class WavFileWriter {
 
 private:
   int sampleCount_;
-  short *buffer_;
-  int bufferSize_;
+  // Buffer in AXI RAM since it has to be reachable by DMA perif
+  __attribute__((aligned(32))) static short buffer_[MAX_SAMPLE_COUNT * 2];
```

Comment:
I thought we were using only the  fixed-width integer types and not int or `short` ?